packages/fuels-types/src/core/sized_ascii_string.rs

```diff
@@ -138,4 +162,20 @@ mod tests {
         // and vice-versa
         assert_eq!("abc", sized_str);
     }
+
+    #[test]
+    fn trim() -> Result<()> {
+        // Every whitespace in those literals is made of a single whitespace
+        let untrimmed = SizedAsciiString::<9>::new(" est abc ".to_string())?;
+
+        assert_eq!("est abc ", untrimmed.to_left_trimmed_string());
+        assert_eq!(" est abc", untrimmed.to_right_trimmed_string());
+        assert_eq!("est abc", untrimmed.to_trimmed_string());
+
+        let padded =
+            SizedAsciiString::<100>::new_with_right_whitespace_padding("victor".to_string())?;
+        assert_eq!(&*format!("{:width$}", "victor", width = 100), padded);
```

Comment:
Hard coding the expectation would be more readable. 
e.g
```suggestion
        let padded =
            SizedAsciiString::<3>::new_with_right_whitespace_padding("victor".to_string())?;
        assert_eq!("victor   ", padded);
```

Also, it avoids using the code we're testing to produce expectations for the test itself. This can lead to a never-failing test.